eng/targets/CSharp.Common.targets

```diff
@@ -27,6 +27,9 @@
         '$(ISBenchmarkProject)' == 'true' OR
         '$(IsSampleProject)' == 'true' OR
         '$(IsMicrobenchmarksProject)' == 'true'">$(NoWarn);CA1416</NoWarn>
+
+    <!-- Enable .NET code style analysis during build for src projects. -->
+    <EnforceCodeStyleInBuild Condition="'$(EnforceCodeStyleInBuild)' == ''">true</EnforceCodeStyleInBuild>
```

Comment:
@BrennanConroy that's what this enables these rules to run on the CLI.

Reply:
This fixes them when it finds them?

Reply:
No, just makes it appear as errors. I used VS to apply the code fix. But presumably once we go thru a few passes of enabling and fixing warnings for individual rules, it's not going to be a big issue. Building in the IDE and on the CLI should produce these errors and they would be localized to new code.

Reply:
> I used VS to apply the code fix.

How long did this take you? Did you have to go project by project?

Reply:
I tried applying it on the sln file but the fixes wouldn't stick. I was however able to get pretty much all of it covered by opening the MVC and Middleware slnfs, and having it load all transitive dependencies. There were a few strays that the build would catch, but not many.

I'm sure there's a better way to go about it, but it's not super painful.